ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -148,6 +158,16 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.neighbors[neighbor].update_source is arista.avd.defined %}
    neighbor {{ neighbor }} update-source {{ router_bgp.neighbors[neighbor].update_source }}
 {%         endif %}
+{%         if router_bgp.neighbors[neighbor].rib_in_pre_policy_retain.enabled is arista.avd.defined(true) %}
```

Comment:
Please, render that after BFD, so we are closer to how it should be printed:
```
   neighbor 192.168.0.8 peer group IPv4-UNDERLAY-PEERS
   neighbor 192.168.0.8 remote-as 65100
   neighbor 192.168.0.8 update-source Loopback1
   neighbor 192.168.0.8 bfd
   neighbor 192.168.0.8 description POD1_SPINE01_Ethernet1
   neighbor 192.168.0.8 rib-in pre-policy retain
   neighbor 192.168.0.8 route-map test out
```

Reply:
fixed